### ISSUE TITLE
Banner is incorrect in the last version

### ISSUE THREAD

**Issue by avysk** [ocaml-community/utop#492]
After upgrading utop via opam to version 2.15.0 and running it, I see

"Welcome to utop version %%VERSION%% (using OCaml version 5.2.1)!"

**Comment by Octachron** [comment]
Indeed, I missed that the release archive was not the source archive of the repository. I will try to fix this shortly.
Sorry!

**Comment by avysk** [comment]
No worries and no hurry, this is just a cosmetic defect; utop is still incredibly useful even with an incorrect banner :)

**Comment by BenitoEck** [comment]
I was wondering about this... Thought I had messed up my installation.

**Comment by yannl35133** [comment]
I stumbled onto this as well; the issue is fixed and the PR for opam-repository is [here](https://github.com/ocaml/opam-repository/pull/27239).
